pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "argotlunar";
+  version = "2.1.0";
+
+  src = pkgs.fetchgit {
+    url = "https://github.com/mourednik/argotlunar";
+    rev = "1527d5dd753a7af32600ac677337469bbf1d40cd";
+    sha256 = "sha256-U20ZUFrck/Y3GwqEnPZewnUN1YIFucgNsJJTMtWY3wE=";
+  };
+
+  buildInputs = [
```

Comment:
This needs to be split into `nativeBuildInputs` and `buildInputs`.

The short summary is that `nativeBuildInputs`  are the programs that need to run as part of the build process, whereas `buildInputs` are the actual runtime dependencies of the software you're packing. This matters a lot when cross compiling, where the two might be for different operating systems or different architectures!

Here I think `pkg-config` and `cmake` should be `nativeBuildInputs`.

Reply:
Ah, thanks! Let me fix it.